server/db/migrator.js

```diff
@@ -43,18 +43,25 @@ var migrator = function (patchLevel) {
     } else {
       pgpatcher(client, level, {dir: __dirname + '/migrations'}, function onPatched(err, result) {
```

Comment:
going back, I also realized that we weren't cleanly handling the migration down to patch level 0, where there is no postgres `user_pages` table to extract from. Added a guard clause against that case.